.github/ISSUE_TEMPLATE/custom_product/tech-day.md

```diff
@@ -7,36 +7,51 @@ labels: ['Tech Day', 'Area:Tech']
 
-If you see a non-compliant element but you didn't get a chance to work on it, please create the corresponding ticket, attach it to the next fixVersion of the codebase and link it in this ticket.
+When creating such tickets, try to provide details about complexity of such an implementation. These elements play a role in our capacity to prioritize work.
 
-Please fill the checklist available in this ticket, priorities are available as a guideline as to what we consider more or less important for each of the tech areas:
+## About priorities
 
-🚨 Indicates a required item, to be looked at during the day
-🔝 Indicates a top priority item
-🟠 Indicates a medium priority item
-🙏 Indicates a low priority item
+* 🚨 Indicates a required item, to be looked at
+* 🔝 Indicates a top priority item
+* 🟠 Indicates a medium priority item
+* 🙏 Indicates a low priority item
 
-## Tech day checklist
+## Checklist
 
-This checklist is focused on a classic Jahia repository (module, app)
+This checklist is there to help you but is not exaustive, if some items are not relevant or should be added, [please request a change](https://github.com/Jahia/.github/blob/master/.github/ISSUE_TEMPLATE/custom_product/tech-day.md).
 
 ### General
-- [ ] 🚨 I reviewed all OPEN TECH tickets created for that codebase (using fixVersion=codebase-X.Y.Z-SNAPSHOT)
+- [ ] 🚨 I reviewed all OPEN TECH tickets created for that codebase (using codebase-X.Y.Z milestone)
 - [ ] 🚨 I reviewed older tech day tickets for that codebase
+- [ ] 🟠 Standards have been discussed in a tech kumite in the past semester
+- [ ] 🙏 Module's license is up-to-date (see https://github.com/Jahia/open-source/blob/master/README.md#licenses)
+### Dependency management
+- [ ] 🔝 I've identified the process/tools to handle dependency updates (ex: [renovate](https://jahia-confluence.atlassian.net/wiki/spaces/PR/pages/2071358/3rd-party+libraries+-+Ref+ISPOL08.A14024#%5BinlineExtension%5DRenovate))
+- [ ] 🟠 Ensure licenses used by the libraries are [Jahia compliant](https://jahia-confluence.atlassian.net/wiki/spaces/PR/pages/2068350/License+check+-+Ref+ISSOP08.A14020)
+- [ ] 🙏 Remove unused libraries
+### Jahia Modules
+- [ ] 🚨 If the codebase is a module shipped with the distribution, the latest version with changes is configured in jahia-pack ([core](https://github.com/Jahia/jahia-pack-private/blob/master/core-modules/pom.xml) or [additional-modules](https://github.com/Jahia/jahia-pack-private/blob/master/additional-modules/pom.xml))
+### Codebase management
+- [ ] 🔝 No warnings or errors are present when building the module locally or on GitHub Actions
+- [ ] 🟠 I reviewed opportunities to remove dead/unused/unreachable code
+- [ ] 🟠 No code smell on [Sonarqube](https://sonarqube.jahia.com/projects) for the module
+- [ ] 🙏 I reviewed opportunities to merge codebases within the repository
```

Comment:
Suggestion: check to make sure dependencies (and appropriate version if needed) are declared in jahia-depends

Reply:
Added an item here: https://github.com/Jahia/.github/pull/81/files#diff-51cbdc8459eee46896ee477ef9880ed2216b827d1590baef88431b78385dc3f5R50